auth-tokens/src/main/java/com/palantir/tokens/auth/UnverifiedJsonWebToken.java

```diff
@@ -159,12 +159,6 @@ private static String decodeUuidBytes(byte[] bytes) {
         @JsonProperty("sid")
         private byte[] sid;
 
-        /**
-         * Indicates this token's expiry (only for session tokens, otherwise null).
-         */
-        @JsonProperty("exp")
-        private Long exp;
```

Comment:
I thought there was a reason we defined this, but we disable failure on unknown properties so it should be okay. I think it's worth retaining a comment at least.

Reply:
Surely we should keep hold of this just so that it round trips?

Reply:
I thought this type was only used for deserialization, not serialization, but I may be remembering incorrectly.

Reply:
Otherwise we should add a test ;-)

Reply:
Doesn't look like it's serialized anywhere, and the class is private